gum/backend-x86/gumstalker-x86.c

```diff
@@ -2801,9 +2805,8 @@ gum_exec_block_virtualize_branch_insn (GumExecBlock * block,
   else if (op->type == X86_OP_MEM)
   {
 #if GLIB_SIZEOF_VOID_P == 4 && defined (HAVE_WINDOWS)
-    if (ctx->stalker->wow64_transition_address != NULL &&
-        op->mem.disp == GPOINTER_TO_UINT (
-            ctx->stalker->wow64_transition_address) &&
+    if (ctx->stalker->ntdll_wow64_transition_address != NULL &&
+        (op->mem.disp == GPOINTER_TO_UINT (ctx->stalker->ntdll_wow64_transition_address) || (ctx->stalker->kernel32_wow64_transition_address != NULL && op->mem.disp == GPOINTER_TO_UINT (ctx->stalker->kernel32_wow64_transition_address))) &&
```

Comment:
Exceeds 80 cols. Style is documented [here](https://github.com/frida/frida/blob/master/CONTRIBUTING.md).

Reply:
Fixed, also I think this is the more correct way